.github/workflows/create-release.yml

```diff
@@ -27,24 +27,27 @@ jobs:
             arch: x86_64
             ext: deb
             protoc-arch: linux-x86_64
-          - target: x86_64-apple-darwin
-            os: macos-latest
-            arch: x86_64
-            ext: pkg
-            protoc-arch: osx-x86_64
-          - target: aarch64-apple-darwin
-            os: macos-latest
-            config-file: fpm/osx.fpm
-            arch: aarch64
-            ext: pkg
-            protoc-arch: osx-aarch_64
+          # - target: x86_64-apple-darwin
```

Comment:
Do you want to try these with the TOML file change and see if they work now? Would be nice to have them but if they don't and it takes longer than a few minutes to figure out, then just Linux is totally fine, since that is what we need for CI in `rust-ceramic`.

Reply:
Yeah, this seems to work now! 🎉